src/gui/qgsmetadatawidget.h

```diff
@@ -43,16 +43,50 @@ class GUI_EXPORT QgsMetadataWidget : public QWidget, private Ui::QgsMetadataWidg
 
     /**
      * Constructor for the wizard.
+     * \note
+     * For use with Layer. Sets mMetadata if pointer is valid
```

Comment:
"For use with a source \layer. This constructor automatically sets the widget's metadata() if the \a layer pointer is valid."

(See note below about not including private variables in the public docs)

Reply:
done